cli/lib/upgrade.go

```diff
@@ -39,7 +37,7 @@ func initUpgradeCommand() *cobra.Command {
 		Long: "This command ungrades all installed libraries to the latest available version." +
```

Comment:
(ultra-nitpicking)
There is a typo here: `ungrades` that should be `upgrades`.
Plus this string should be joined to the next one with a space because I have it printed in my terminal like this:
`This command ungrades all installed libraries to the latest available version.To upgrade a single library use the 'install' command.`
instead of be like this:
`This command ungrades all installed libraries to the latest available version. To upgrade a single library use the 'install' command.`
In addition, in order to differentiate from the `lib install` command mentioned in the message, we could add something like `To upgrade multiple libraries in a single command add the name of the libraries as parameters and separate them by a space` to clearly state that update multiple libraries is possible